conf/services/get-a-goods-movement-reference.yml

```diff
@@ -3,37 +3,18 @@ serviceHeaderName: Get a goods movement reference
 serviceDescription: >
   You need a goods movement reference to move goods between Great Britain and
   countries in the European Union and Northern Ireland. This service is part of
-  the goods vehicle movement service (GVMS).
+  the Goods Vehicle Movement Service (GVMS).
 serviceDomain: www.tax.service.gov.uk
 serviceUrl: /goods-movement-system/dashboard
 contactFrontendServiceId: GaGMR
 complianceStatus: partial
 accessibilityProblems:
-  - on the Check direction and check your answers pages the screen reader reads
-    'change' . It should read "change" and give the context of what the
-    user would be changing, for example "change direction".
-  - when a screen reader gets to the grey footer, it reads support links but
-    there are none there
-  - on the task list, customs declaration reference and safety and security
-    number pages there are missing ARIA labels on some fields.
+  - Some users may have difficulty negotiating the ‘Enter a vehicle registration number page’ as the input box is missing a label. Screen readers won’t announce this input box so entering a vehicle registration number will be difficult.
 milestones:
-  - date: 2021-03-31
+  - date: 2021-09-30
     description: >-
-      On the check direction and check your answers pages the screen reader
-      reads 'change' . It should read "change" and give the context of what
-      the user would be changing. e.g. "change direction". This doesn't meet
-      WCAG success criterion 2.4.4 (link purpose).
-  - date: 2021-03-31
-    description: >-
-      When a screen reader gets to the grey footer, it reads "support links" but
-      there are none there. This does not meet WCAG success criterion 1.3.1
-      (info and relationships).
-  - date: 2021-03-31
-    description: >-
-      On the task list, customs declaration reference and safety and security
-      number pages there are missing ARIA labels on some fields. This doesn't
-      meet WCAG success criterion 3.3.2 (labels and instructions).
-serviceLastTestedDate: 2019-08-19
+      There is a missing label on the ‘Enter the vehicle registration number’ page, so it has no accessible name. This is a WCAG 2.1 violation: Understanding Success Criterion 4.1.2 Name, Role, Value.
+serviceLastTestedDate: 2020-11-23
 statementVisibility: public
-statementCreatedDate: 2020-11-16
-statementLastUpdatedDate: 2020-11-16
+statementCreatedDate: 2021-08-13
```

Comment:
do not change the created date I thought that would be the same?

Reply:
I thought it would be the same as well so originally left it untouched, but then one of the dates in the rendered statement doesn't match the document version of the statement attached to the ticket. It's one of a few things I am going to ask Mike about.

Reply:
Have reverted to original creation date and confirmed with Mike that this is correct. Will add a note to the ticket that the date in the doc is wrong.